app/models/my_module_status_flow.rb

```diff
@@ -21,4 +21,18 @@ def initial_status
   def final_status
     my_module_statuses.left_outer_joins(:next_status).find_by('next_statuses_my_module_statuses.id': nil)
   end
+
+  def self.ensure_default
+    return if MyModuleStatusFlow.global.any?
+
+    sf = MyModuleStatusFlow.create(name: Extends::DEFAULT_FLOW_NAME, visibility: :global)
+    prev_id = nil
+    Extends::DEFAULT_FLOW_STATUSES.each do |s|
+      s = MyModuleStatus.create(my_module_status_flow: sf,
+                                name: s[:name],
+                                color: s[:color],
+                                previous_status_id: prev_id)
+      prev_id = s.id
```

Comment:
will you add here support for the creation of implication, condition, and consequences